samples/unit-tests/boost/general/demo.css

```diff
@@ -0,0 +1,5 @@
+#container {
+    width: 600px;
+    height: 350px;
```

Comment:
Do we need height=350px? Or can we use just 400px (default)? I'm asking this question, because Karma does not load `demo.css` files, meaning the chart will have different height in utils and Karma

Reply:
Absolutely not, I just copied it from another demo. I'll delete the height property :)